consensus/crdt/consensus.go

```diff
@@ -257,6 +257,11 @@ func (css *Consensus) setup() {
 		return
 	}
 	css.state = clusterState
+
+	if css.config.TrustAll {
+		logger.Info("'trust all' mode enabled. Any peer in the cluster can modify the pinset.")
```

Comment:
Didn't you want to use `logger.Warn` ?

Reply:
No, because I don't want to warn for something that is a default setting. I want to INFOrm the user. Commit message not very accurate in this regard.